modules/calendar/cog.py

```diff
@@ -264,24 +265,43 @@ async def update_event(self, ctx: commands.Context, *, args=None):
 			calendar = self.finder.get_calendar(ctx.author, calendar_name)
 		except (ClassRoleError, ClassParseError) as error:
 			raise FriendlyError(error.args[0], ctx.channel, ctx.author)
+		# loading message
+		response = await ctx.send(embed=embed_success("🗓 Searching for events..."))
+		# get a list of upcoming events
 		events = self.calendar_service.fetch_upcoming(calendar.id, 50, query)
-		if len(events) == 0:
+		num_events = len(events)
```

Comment:
What was the point of this? I'm pretty sure len(events) is O(1), there's no point storing it in a variable

Reply:
I feel it looks cleaner to use a variable as an array limit instead of a function call when doing this:

```python
self.number_emoji[:num_events]
```

Can change it if you want, though.